### PR TITLE
[docs] Issue 19496

### PR DESCRIPTION
**Pull request by ddhodge**
#19496

@netlify /stable/yugabyte-platform/install-yugabyte-platform/install-software/installer/#configuration-options

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 1]
```diff
@@ -327,7 +327,7 @@ You can set the following YBA Installer configuration options.
 
 | Option | Description |
 | :--- | :--- |
-| `installRoot` | Location where YBA is installed. Default is `/opt/yugabyte`. |
+| `installRoot` | Location where YBA is installed. Default is `/opt/yugabyte`. Note: on CentOS 7, this parameter cannot exceed 16 characters due to CentOS 7 python script shebang limits. |
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:330]:
```suggestion
| `installRoot` | Location where YBA is installed. Default is `/opt/yugabyte`. <br> Note: on CentOS 7, this parameter cannot exceed 16 characters due to CentOS 7 python script shebang limits. |
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 2]
```diff
@@ -323,7 +323,7 @@ You can set the following YBA Installer configuration options.
 
 | Option | Description |
 | :--- | :--- |
-| `installRoot` | Location where YBA is installed. Default is `/opt/yugabyte`. |
+| `installRoot` | Location where YBA is installed. Default is `/opt/yugabyte`. Note: on CentOS 7, this parameter cannot exceed 16 characters due to CentOS 7 python script shebang limits. |
```
Comment (aishwarya24) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:326]:
```suggestion
| `installRoot` | Location where YBA is installed. Default is `/opt/yugabyte`. <br> Note: on CentOS 7, this parameter cannot exceed 16 characters due to CentOS 7 python script shebang limits. |
```

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Review by chidmuthu — CHANGES_REQUESTED** [review]
Actually sorry upon further discussion we need to fix this on the product side so don't need this documentation clarification. Can abandon this

**Pull request closed without merging**